data/counter.csv

```diff
@@ -1,5 +1,8 @@
 Command Name,Frequency
+DeletePatientTaskCommand,5
 DeletePatientCommand,1
-ListTasksCommand,1
+ListPatientsCommand,1
+ListTasksCommand,2
 AddStandardTaskCommand,6
 AddPatientCommand,3
+FindPatientTaskCommand,2

```

Comment:
You should not commit csv. Or else everyone will need to resolve conflicts for this.

Reply:
noted